addons/account/tests/test_account_journal.py

```diff
@@ -166,25 +166,6 @@ def test_account_journal_duplicates(self):
 
         self.assertEqual(sorted(new_journals.mapped("code")), ["GEN1", "OD_BL"], "The journals should be set correctly")
 
-    def test_archive_used_journal(self):
```

Comment:
Imho, you could leave this test with the opposite assert, so that we won't have a regression in future